packages/integrations/image/README.md

```diff
@@ -28,11 +28,11 @@ The `astro add` command-line tool automates the installation for you. Run one of
    
 ```sh
 # Using NPM
-npx astro add image
+npm run astro add image
 # Using Yarn
 yarn astro add image
 # Using PNPM
-pnpx astro add image
+pnpm run astro add image
```

Comment:
Since we use `yarn astro` for yarn, perhaps we can do `pnpm astro` too? it's supported since pnpm 7, which should be commonly used now.